test/client.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+
+import Image from '../client/src/Image.jsx';
+
+describe('dummy test to make sure jest is working', () => {
+  test('checks if true is true', () => {
+    expect(true).toBe(true);
+  });
+});
+
+describe('dummy tests to make sure enzyme is working', () => {
+  test('checks if shallowly rendered html is retrieved using .html()', () => {
+    const myImage = shallow(<div>test</div>);
+    expect(myImage.html()).toEqual('<div>test</div>');
+  });
+
+
+});
+
+describe('it should render the correct image', () => {
+  test('should render an component of type Image', () => {
+    const myImage = shallow(<Image productId={0} />);
+    const instance = myImage.instance();
+    expect(instance).toBeInstanceOf(Image);
+  });
+
+  test('checks if an image is rendered', () => {
+    const myImage = shallow(<Image productId={0} />);
+    const instance = myImage.instance();
+    expect(instance).toBeInstanceOf(Image);
+  });
```

Comment:
these two seem to be duplicates

Reply:
oh you're right! good catch. Thank you! Will delete one of them and then merge.